src/website/views/components/data-table/_examples.pug

```diff
@@ -0,0 +1,2712 @@
+            span Last Login
+      .chi-data-table__body
+        .chi-data-table__row
+          .chi-data-table__cell(data-label="Name")
+            div Name 1
+          .chi-data-table__cell(data-label="ID")
+            div name-1
+          .chi-data-table__cell(data-label="Last Login")
+            div Dec 18, 2020 3:26 PM
+        .chi-data-table__row
+          .chi-data-table__cell(data-label="Name")
+            div Name 2
+          .chi-data-table__cell(data-label="ID")
+            div name-2
+          .chi-data-table__cell(data-label="Last Login")
+            div Dec 18, 2020 2:38 AM
+        .chi-data-table__row
+          .chi-data-table__cell(data-label="Name")
+            div Name 3
+          .chi-data-table__cell(data-label="ID")
+            div name-3
+          .chi-data-table__cell(data-label="Last Login")
+            div Nov 5, 2020 10:15 AM
+      .chi-data-table__footer
+        nav.chi-pagination(role='navigation', aria-label='Pagination')
+          .chi-pagination__content
+            .chi-pagination__start
+              .chi-pagination__results
+                span.chi-pagination__label 240 results
+              .chi-pagination__page-size
+                .chi-dropdown
+                  button.chi-button.-flat.-px--1.chi-dropdown__trigger#pagesize-3 20
+                  .chi-dropdown__menu.-w--xs
+                    a.chi-dropdown__menu-item.-active(href='#') 20
+                    a.chi-dropdown__menu-item(href='#') 40
+                    a.chi-dropdown__menu-item(href='#') 60
+                    a.chi-dropdown__menu-item(href='#') 80
+                    a.chi-dropdown__menu-item(href='#') All
+                span.chi-pagination__label per page
+            .chi-pagination__center
+              .chi-pagination__button-group.chi-button-group
+                a(class=`chi-button -icon -flat`, href='#', aria-label='Previous page')
+                  .chi-button__content
+                    i(class="chi-icon icon-chevron-left", aria-hidden='true')
+                a(class=`chi-button -flat`, href='#') 1
+                div(class=`chi-button -flat -disabled`) ...
+                a(class=`chi-button -flat`, href='#') 3
+                a(class=`chi-button -flat`, href='#') 4
+                a(class=`chi-button -flat -active`, href='#') 5
+                a(class=`chi-button -flat`, href='#') 6
+                a(class=`chi-button -flat`, href='#') 7
+                div(class=`chi-button -flat -disabled`) ...
+                a(class=`chi-button -flat`, href='#') 12
+                a(class=`chi-button -icon -flat`, href='#', aria-label='Next page')
+                  .chi-button__content
+                    i(class="chi-icon icon-chevron-right", aria-hidden='true')
+            .chi-pagination__end
+              .chi-pagination__jumper
+                label.chi-pagination__label(for="jumper-input-1") Go to page:
+                input(type='text', class='chi-input', id='jumper-input-1')
```

Comment:
Seeing a warning in the page as the ID `jumper-input-1` is used in multiple examples